pipelines/cellpainting/cpd_max_projection_pipeline.wdl

```diff
@@ -41,7 +46,7 @@ workflow cpd_max_projection_distributed {
       input:
         image_directory =  images_directory,
         illum_directory = "/illum",  # default
-        load_data_csv = images_directory + "/load_data.csv",
+        load_data_csv = load_data_directory + "/load_data.csv",
```

Comment:
If you think the above is a good idea, it would also be changed here

Reply:
Yes. done :)